src/admin/resolvers/mutations/ScheduledItem.integration.ts

```diff
@@ -204,6 +210,22 @@ describe('mutations: ScheduledItem', () => {
         otherReturnedApprovedItemProps
       );
 
+      // check authors
+      // note that approvedItemAuthors does not go through our graphql interface,
+      // so it has *all* db properties, including externalId and approvedItemId.
+      // these properties are *not* present in authorsReturned, so we need to do
+      // a custom comparison
+      if (approvedItemAuthors) {
+        const approvedItemAuthorsMapped = approvedItemAuthors.map((aia) => {
+          return {
+            name: aia.name,
+            sortOrder: aia.sortOrder,
+          };
+        });
+
+        expect(approvedItemAuthorsMapped).to.deep.equal(authorsReturned);
```

Comment:
jest has this built in: https://jestjs.io/docs/expect#expectobjectcontainingobject